TypeCobol.LanguageServer/Completion Factory/CompletionFactory.cs

```diff
@@ -42,9 +44,13 @@ public static List<CompletionItem> GetCompletionPerformParagraph(FileCompiler fi
                 }
             }
 
-            if (pargraphs != null)
+            if (paragraphs != null)
             {
-                completionItems.AddRange(pargraphs.Select(para => new CompletionItem(para.Name) { kind = CompletionItemKind.Reference }));
+                completionItems.AddRange(paragraphs.Select(para => new CompletionItem(para.Name) { kind = CompletionItemKind.Reference }));
+            }
+            if (sections != null)
```

Comment:
`GetParagraphs` and `GetSections` cannot return `null` so you can safely remove both null checks

Reply:
Done